### DISCUSSION TITLE
Rancher Desktop 1.8.1

### DISCUSSION THREAD

**Discussion by jandubois** [rancher-sandbox/rancher-desktop#4254] [Archive]
This is the 1.8.1 release of Rancher Desktop, an open source desktop application to bring Kubernetes and container management to Mac, Windows, and Linux.

## This patch release fixes one issue in Rancher Desktop 1.8.0

The new experimental deployment profile functionality was broken on Windows. This has been fixed.

There are no other changes in this release compared to 1.8.0

Please check the [Rancher Desktop 1.8.0 release notes](https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.8.0) for a comprehensive list of new features and bug fixes.

## Connect with the developers

- The [issue queue](https://github.com/rancher-sandbox/rancher-desktop/issues)

- [Rancher Users Slack](https://slack.rancher.io/) in the #rancher-desktop channel

## Changelog

The [full version changelog, from v1.8.0, can be found using GitHub compare](https://github.com/rancher-sandbox/rancher-desktop/compare/v1.8.0...v1.8.1) and the details of the release can be found in the [v1.8.1 milestone](https://github.com/rancher-sandbox/rancher-desktop/milestone/31?closed=1).

<hr /><em>This discussion was created from the release <a href='https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.8.1'>Rancher Desktop 1.8.1</a>.</em>

**Comment by a-magdy** [comment]
Hi there, the asset: Rancher.Desktop-1.8.1.arch64.dmg has a missing (a) in aarch64 which breaks the download link you have on your homepage: https://rancherdesktop.io/ for Mac silicon

Link from homepage vs link on github:

https://github.com/rancher-sandbox/rancher-desktop/releases/download/v1.8.1/Rancher.Desktop-1.8.1.aarch64.dmg
https://github.com/rancher-sandbox/rancher-desktop/releases/download/v1.8.1/Rancher.Desktop-1.8.1.arch64.dmg

Not sure if you want to update the link template in this file: https://github.com/rancher-sandbox/rancherdesktop.io/blob/main/content/_index.html#L26
or update the asset name in the release on GitHub

Thanks for your effort <3

**Reply by jandubois** [reply]
Thank you @a-magdy for reporting; I've renamed the file and created a corresponding sha512sum for it, so everything should be fixed now.
